commit 21320700798da9f7da6b50b16c44e1305f14f2bc
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Mar 17 00:48:23 2017 +0000

    PCI: Decouple IORESOURCE_ROM_ENABLE and PCI_ROM_ADDRESS_ENABLE
    
    [ Upstream commit 7a6d312b50e63f598f5b5914c4fd21878ac2b595 ]
    
    Remove the assumption that IORESOURCE_ROM_ENABLE == PCI_ROM_ADDRESS_ENABLE.
    PCI_ROM_ADDRESS_ENABLE is the ROM enable bit defined by the PCI spec, so if
    we're reading or writing a BAR register value, that's what we should use.
    IORESOURCE_ROM_ENABLE is a corresponding bit in struct resource flags.
    
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Reviewed-by: Gavin Shan <gwshan@linux.vnet.ibm.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1426_21320700798da9f7da6b50b16c44e1305f14f2bc.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1426_21320700798da9f7da6b50b16c44e1305f14f2bc.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
